bin/select_targets

```diff
@@ -50,8 +52,18 @@ if ns.check:
 else:
     targets = select_targets(infiles, numproc=ns.numproc, verbose=ns.verbose,
                              qso_selection=ns.qsoselection, sandbox=ns.sandbox, FoMthresh=ns.FoMthresh, Method=ns.Method)
-    if ns.starmask:
-        targets = mask_targets(targets, instarmaskfile=ns.starmask, nside=nside)
+    if ns.mask:
+        targets = mask_targets(targets, instarmaskfile=ns.mask, nside=nside)
+
+    if ns.skies:
+        skies = select_skies(infiles,numproc=ns.numproc)
+        ntargs = len(targets)
+        nskies = len(skies)
+        nobjs = ntargs + nskies
+        targs = np.empty(nobjs, dtype=targets.dtype)
+        targs[0:ntargs] = targets
+        targs[ntargs:nobjs] = skies
+        targets = targs.copy()
```

Comment:
I don't think this copy is necessary.  Simply `targets = targs` would allow the I/O to proceed without having 2 copies in memory (python garbage collection might not get around to cleaning it up anyway before finishing, but at least this would give it the option of doing so if needed).

Reply:
Agreed. I've removed the `copy()`